WebAPI.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,89 @@
+using System.Text.Json.Serialization;
+using RestSharp;
+using WebAPI.Infrastructure;
+using WebAPI.Infrastructure.RequestsResponses;
+
+namespace WebAPI.Tests;
+
+public class Tests
+{
+    private RestApiClient _restClient;
+    private int _id;
+
+    [SetUp]
+    public async Task Setup()
+    {
+        _restClient = new RestApiClient();
+        await _restClient.Authenticate("admin", "password123");
+    }
+
+    [Test, Order(1)]
+    public async Task CreateBooking()
+    {
+        var dates = new Bookingdates 
+        { 
+            checkin = DateOnly.Parse("2018-01-01"),
+            checkout = DateOnly.Parse("2019-01-01") 
+        };
+        
+        var meta = new BookingMetaData
+        {
+            firstname = "Charles",
+            lastname = "Leclerc",
+            totalprice = 12,
+            depositpaid = true,
+            bookingdates = dates,
+            additionalneeds = "empty"
+        };
+        
+        var response = await _restClient.CreateBooking(meta);
+
+        _id = response.Data.bookingid;
+        
+        Assert.AreEqual(System.Net.HttpStatusCode.OK, response.StatusCode);
```

Comment:
It is not enough to check just status code. Please use Booking - GetBookingIds endpoint to get all booking and check that your booking is present among them. Use fluent assertions for this check .Should().Contains(......)